tasks/prerequisite.yml

```diff
@@ -25,4 +25,5 @@
   package: name={{ item }} state=latest
   with_items:
     - hp-scripting-tools
+    - hponcfg
```

Comment:
We do not have hponcfg in the repo we use (fwpp [1]).

So this brings in a dependency on another repo for us, which I'm not a big fan of.

[1]: https://downloads.linux.hpe.com/SDR/project/fwpp/index.html